docker-compose.yaml

```diff
@@ -527,13 +527,13 @@ services:
         volumes:
             - ./postgres-data-current:/var/lib/postgresql/data
             - ./other/postgres-entrypoint.sh:/postgres-entrypoint.sh
-            - ./postgres-data-backup/:/tmp/
+            - ./postgres-data-backup/:/postgres-entrypoint.d/
```

Comment:
is this correct for backup data mappings?

Reply:
I replaced that and worked with /tmp this time :) can you check locally as well?

Reply:
could you replace with `/tmp/` then?